packages/@o3r/testing/package.json

```diff
@@ -139,6 +139,12 @@
     "@schematics/angular": {
       "optional": true
     },
+    "pixelmatch": {
```

Comment:
These deps are used by modules exposed in sub-entry only?
If the module using these deps are exposed at barrel level, you won't be able to turn them optional :S